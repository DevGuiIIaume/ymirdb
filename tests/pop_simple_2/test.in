set a -1 0 1
pop a
get a
del a
pop a
bye
