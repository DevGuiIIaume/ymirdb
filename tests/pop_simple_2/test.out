> ok

> -1

> [0 1]

> ok

> no such key

> bye
